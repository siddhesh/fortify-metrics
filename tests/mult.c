typedef __SIZE_TYPE__ size_t;

size_t
__attribute__ ((access, __read_write__, 1, 2))
f1 (const char *c, size_t s)
{
  return __builtin_object_size (c, 0);
}

size_t
f2 (size_t s)
{
  char *obj = __builtin_malloc (s);

  return __builtin_object_size (s, 0);
}

size_t
f3 (int cond)
{
  char buf[24];
  char *obj = buf;

  if (cond)
    obj = malloc (48);;

  return __builtin_object_size (s, 0);
}
