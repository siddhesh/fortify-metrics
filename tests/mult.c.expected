fmetrics:: 1:3
